src/main/java/io/github/cottonmc/cotton/gui/widget/WScrollBar.java

```diff
@@ -109,7 +120,7 @@ public boolean canResize() {
 	public boolean canFocus() {
 		return true;
 	}
-
+	@Deprecated
 	private static void drawBeveledOutline(DrawContext context, int x, int y, int width, int height, int topleft, int bottomright) {
```

Comment:
Is this method still used somewhere? If yes, it shouldn't be deprecated, and if not, it should be removed.